src/core_plugins/kibana/public/dashboard/panel/__snapshots__/dashboard_panel.test.js.snap

```diff
@@ -10,6 +10,7 @@ exports[`DashboardPanel matches snapshot 1`] = `
   >
     <div
       class="panel-heading"
+      data-test-subj="dashboardPanelHeading-"
```

Comment:
This looks like it's missing info after the `-` was it supposed to have some value there, like the title? Probably not important.

Reply:
There isn't a title in the test data for this, but I can add it in so it tests that the title is there and that it doesn't look like a mistake.